Source/Lib/C_DEFAULT/EbComputeSAD_C.c

```diff
@@ -192,7 +192,7 @@ void SadLoopKernel(
 			{
 				for (x = 0; x < width; x++)
 				{
-					sad += EB_ABS_DIFF(src[y*srcStride + x], ref[xSearchIndex + y*refStride + x]);
+					sad += EB_ABS_DIFF(src[y*srcStride + x], ref[xSearchIndex + (ySearchIndex+y)*refStride + x]);
```

Comment:
looks good to me, 
did you do a unit test checking the output of the function before and after your change compared to the optimized version ?